src/v2/Components/UserSettings/TwoFactorAuthentication/helpers.ts

```diff
@@ -0,0 +1,17 @@
+import { getClientParam } from "v2/Utils/getClientParam"
+import sanitizeRedirect from "@artsy/passport/sanitize-redirect"
+
+export const afterUpdateRedir = () => {
+  const afterUpdateURL = getClientParam("after_update")
+
+  if (afterUpdateURL) {
+    const sanitizedURL = sanitizeRedirect(afterUpdateURL)
+
+    // a '/' means URL is bad
+    if (sanitizedURL !== '/') {
+      return sanitizedURL
+    }
+  }
+
+  return ''
```

Comment:
If I'm following the logic correctly, returning blank here means the `if(redirectTo) ...` checks [above](https://github.com/artsy/force/pull/7626/files#diff-bd0d1fdf420315bc176a724649061dc306c49c7f18d9c2c2feee55904b53b7efR64) will be false, so the non-redirecting modal will display, right?

Reply:
yes, that's the intent, but..yeah, it's kinda confusing.